utils/pigz/Makefile

```diff
@@ -0,0 +1,32 @@
+# from https://zlib.net/pigz/
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pigz
+PKG_VERSION:=2.4
+PKG_RELEASE:=4
+PKG_BUILD_DIR:=$(BUILD_DIR)/pigz-$(PKG_VERSION)
```

Comment:
This shouldn't be necessary as it is same as:
PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)

Reply:
ok